driver-core/src/main/java/com/datastax/driver/core/RequestHandler.java

```diff
@@ -58,6 +59,7 @@
     private final io.netty.util.Timer scheduler;
 
     private volatile List<Host> triedHosts;
+    private volatile Statement actualStatement;
```

Comment:
`actualStatement` could be removed after https://github.com/datastax/java-driver/pull/540/files#r45767990